src/main/java/uk/gov/pay/api/filter/RateLimiterFilter.java

```diff
@@ -62,12 +62,12 @@ private void setTooManyRequestsError() throws IOException {
 
     @Override
     public void filter(ContainerRequestContext requestContext) throws IOException {
-        final String authorization = requestContext.getHeaderString("Authorization");
         final String method = requestContext.getMethod();
 
         String accountId = getAccountId(requestContext);
+        RateLimiterKey key = RateLimiterKey.from(requestContext, accountId);
```

Comment:
I know I could calculate it in the RateLimiterKey, but that would introduce the whole lot of changes (it can be done separately though)